docs/install.md

```diff
@@ -448,13 +463,16 @@ verify that the node labels have been configured correctly
 $ kubectl get nodes --show-labels
 ```
 
-The command output must indicate that every node with PMEM has these two labels:
+The command output must indicate that every node with PMEM has at least two labels:
 ``` console
 pmem-csi.intel.com/node=<NODE-NAME>,storage=pmem
 ```
 
-If **storage=pmem** is missing, label manually as described above. If
-**pmem-csi.intel.com/node** is missing, then double-check that the
+**storage=pmem** is the label that has to be added manually as
+described above.  When using NFD, the node should have the
+`feature.node.kubernetes.io/memory-nv.dax: true` label.
```

Comment:
Replacing ';' with '=' is most appropriate here.

Reply:
Fixed.